### PR TITLE
migration: Add case to test paused vm

### PR DESCRIPTION
**Pull request by cliping**
VIRT-298232 - [VM migration] migrate paused vm

### CODE REVIEW

**Comment by cliping** [comment]
Test result:
 (1/4) type_specific.io-github-autotest-libvirt.migration.migrate_vm_in_various_status.migrate_paused_vm.migration_succeeds.precopy.p2p: PASS (206.36 s)
 (2/4) type_specific.io-github-autotest-libvirt.migration.migrate_vm_in_various_status.migrate_paused_vm.migration_succeeds.precopy.non_p2p: PASS (207.73 s)
 (3/4) type_specific.io-github-autotest-libvirt.migration.migrate_vm_in_various_status.migrate_paused_vm.cancel_migration.precopy.p2p: PASS (203.55 s)
 (4/4) type_specific.io-github-autotest-libvirt.migration.migrate_vm_in_various_status.migrate_paused_vm.cancel_migration.precopy.non_p2p: PASS (204.52 s)

**Review by nanli1 — APPROVED** [review]

File: libvirt/tests/cfg/migration/migrate_vm_in_various_status/migrate_paused_vm.cfg [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    image_convert = 'no'
+    server_ip = "${migrate_dest_host}"
+    server_user = "root"
+    server_pwd = "${migrate_dest_pwd}"
+    migrate_desturi_port = "16509"
+    migrate_desturi_type = "tcp"
+    virsh_migrate_desturi = "qemu+tcp://${migrate_dest_host}/system"
+    start_vm = "yes"
+    simple_disk_check_after_mig = "no"
+
```
Comment (chunfuwen) [inline libvirt/tests/cfg/migration/migrate_vm_in_various_status/migrate_paused_vm.cfg:25]: maybe remove this blank line

Comment (cliping) [reply libvirt/tests/cfg/migration/migrate_vm_in_various_status/migrate_paused_vm.cfg]: Updated, thanks.

**Review by chunfuwen — APPROVED** [review]
lgtm

**Pull request merged**